l10n_fi_banks/__init__.py

```diff
@@ -0,0 +1 @@
+import models
```

Comment:
This import is breaking the build: see https://travis-ci.org/OCA/l10n-finland/jobs/227352221#L399